internal/fromproto6/readdatasource_test.go

```diff
@@ -46,6 +47,8 @@ func TestReadDataSourceRequest(t *testing.T) {
 		},
 	}
 
+	testClientCapabilities := tfprotov6.ReadDataSourceClientCapabilities{DeferralAllowed: true}
```

Comment:
Nit: For single usage, maybe easier for future travelers to not add the variable indirection